Circular_kernel_2/include/CGAL/Circular_kernel_2/function_objects_polynomial_circular.h

```diff
@@ -1299,15 +1299,15 @@ namespace CircularFunctors {
 
     template<typename F>
     struct result<F(Circular_arc_2)> {
-      typedef typename cpp11::result_of<LK_Compute_squared_radius_2(Circle_2)>::type type;
+      typedef decltype(std::declval<LK_Compute_squared_radius_2>()(Circle_2())) type;
     };
 
```

Comment:
Since we decided to use `auto` or `decltype(auto)` for type-inference, I think all the `result` and specialization of `result` can be removed. It was part of the `result_of` protocol. and no longer used.

Reply:
I see. I will have a look inside it and try to remove all occurrences of the `result`. If I am not mistaken, the `result_of` is only in docs now, at least for kernels.